packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.kt

```diff
@@ -27,6 +27,8 @@ import expo.modules.updates.manifest.UpdateManifest
 // these unused imports must stay because of versioning
 /* ktlint-disable no-unused-imports */
 import expo.modules.updates.UpdatesConfiguration
+import expo.modules.updates.statemachine.UpdatesStateContext
```

Comment:
move to normal imports (this keeps tripping me up, too, and is quite annoying)

Reply:
Done